docs/content/copying-bundles.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Copying Bundles
+description: How to copy a bundle from one registry to another
+---
+
+Porter allows you to copy a bundle, and all associated images, from one registry to another. This is useful when you have restrictions on where you can pull Docker images from or would otherwise like to have control over assets you will be using. Any operation on the copied bundle will utilize these copied images as well. If you'd like to rename something within a registry, Porter also allows you to copy from one bundle tag to another bundle tag inside the same registry.
```

Comment:
When we say all associated images, can we be a bit more clear? Maybe give an example like invocation images and images referenced in the image map (then link to the image map)? I think that would help people understand the scope of what magic is about to occur here.

Otherwise it's easy to fall into thinking that say a image used in a helm chart with automatically get copied too, but it won't unless they do the extra work to make that happen.